package.json

```diff
@@ -144,7 +144,7 @@
           "snackbar",
           "switch",
           "tabs",
-          "textfield",
```

Comment:
I'm really glad we're updating names to fix past typos. So thank you for creating this PR.

But renaming a node module is a bit _too_ breaking...I think this should just be left as `text-field`